src/NzbDrone.Common/Instrumentation/Sentry/SentryTarget.cs

```diff
@@ -94,13 +100,33 @@ private static List<string> GetFingerPrint(LogEventInfo logEvent)
             return fingerPrint;
         }
 
+        private bool IsSentryMessage(LogEventInfo logEvent)
+        {
+            if (logEvent.Properties.ContainsKey("Sentry"))
+            {
+                return logEvent.Properties["Sentry"] != null;
+            }
+
+            if (logEvent.Level >= LogLevel.Error && logEvent.Exception != null)
```

Comment:
Is it really still a sentry message if it doesn't contain the sentry property checked above?

Reply:
If we want it to be, looks like Sonarr wanted to see any log messages with error or fatal level, even if not marked as sentry property. Maybe not bad idea?